File: IdeaProjects/MessageReader/src/main/java/messageReader/MessageReaderConstants.java
package messageReader;

public class MessageReaderConstants {

    public MessageReaderConstants() {
    }

    static final String DATE = "Date";
    static final String TIME = "Time";
    public static final String STATUS = "Status";
    public static final String ENTITY_NAME = "EntityName";

    // For Message Sender
    public static final String FILE_PATH = "..\\MessageReader\\xmlFiles\\";
    public static final String FILE_EXTENSION = ".xml";
    public static final String FILE_PREFIX = "STATUS_0";

    //Channel Constants
    static final String TIMER_PROCESSING_OUTPUT_CHANNEL = "RouterToTimerMonitorChannel";
    static final String STATUS_PROCESSING_OUTPUT_CHANNEL = "RouterToStatusMonitorChannel";

    //Testing constants
    public static final String RESULTS_FILE_NAME = "MessageResults.csv";
    public static final String RESULTS_FILE_PATH = "..\\MessageReader\\results\\";
    public static final String DELIMETER = ",";
    public static final String START_TIME = "StartMessageProcessingTime";
    public static final String END_TIME = "EndMessageProcessingTime";
    public static final String END_CONVERSION_TIME = "EndConversionTime";


}

File: IdeaProjects/ScalaTutorial/ScalaMessageReader/src/main/scala/scalaMessageReader/MessageReaderConstants.scala
package scalaMessageReader

object MessageReaderConstants {
  val ENTITY_NAME = "EntityName"
  val STATUS = "Status"

  val START_TIME = "StartTime"
  val END_TIME = "EndTime"
  val END_CONVERSION_TIME = "EndConversion"

  val DATE = "Date"
  val TIME = "Time"
  val FILE_PATH = "..\\ScalaTutorial\\ScalaMessageReader\\xmlFiles\\"
  val FULL_FILE_PATH = "C:\\Users\\mattg\\IdeaProjects\\ScalaTutorial\\ScalaMessageReader\\xmlFiles\\"
  val FILE_EXTENSION = ".xml"
  val FILE_PREFIX = "STATUS_0"

//  MESSAGE CHANNELS
  val STATUS_PROCESSING_OUTPUT_CHANNEL = "RouterToStatusMonitorChannel"
  val TIMER_PROCESSING_OUTPUT_CHANNEL = "RouterToTimerMonitorChannel"
  val RESULTS_FILE_NAME = "MessageResults.csv"
  val RESULTS_FILE_PATH = "C:\\Users\\mattg\\IdeaProjects\\ScalaTutorial\\ScalaMessageReader\\results\\"
}
